### DISCUSSION TITLE
跳过任意开屏广告和内部弹窗广告

### DISCUSSION THREAD

**Discussion by shenweiyan** [shenweiyan/Digital-Garden#39] [2.2-好玩]
今天去用了一下 [gkd](https://github.com/gkd-kit/gkd)—— 点击跳过任意开屏广告/点击关闭应用内部任意弹窗广告的应用，发现真不错，记录一下遇到的一些问题。

具体使用步骤如下。

## 1. 开启权限

应用安装后，**"主页"** 页面的这几个权限都需要开启（尤其是**无障碍权限**），否则应用无法跳过广告，不起作用。   

<img width=300px src="https://slab-1251708715.cos.ap-guangzhou.myqcloud.com/KGarden/2023/gkd-home.jpg" alt="gkd-home">

华为鸿蒙系统怎么开启无障碍模式（也可以参考：[通过快捷方式启动无障碍功能](https://consumer.huawei.com/cn/support/content/zh-cn15849085/)）： 
   
1. 打开辅助功能，进入手机设置界面，点击【辅助功能】。
2. 打开无障碍，点击【无障碍】。
3. 打开已安装的服务，下拉到页面底部，点击【已安装的服务】。
4. 选择服务，选择要设置的服务【GKD】，点击进入。
5. 打开服务开关，点击服务右侧【开关】按钮。
6. 确定打开。

这里会有一个问题：**鸿蒙手机无障碍打开后，点击清理后台会自动关闭**！参考：[gkd-kit/gkd#201](https://github.com/gkd-kit/gkd/issues/201)

具体可以参考华为官方给出的解决方法——《[华为手机/平板无障碍中的第三方应用的服务开关自动关闭](https://consumer.huawei.com/cn/support/content/zh-cn00410039/)》。

## 2. 更新订阅

应用安装后，**"订阅"** 页面的本地订阅如果没有规则，则需要下拉执行更新一下即可（参考：[gkd-kit/gkd#100](https://github.com/gkd-kit/gkd/issues/100)）。

<img width=300px src="https://slab-1251708715.cos.ap-guangzhou.myqcloud.com/KGarden/2023/gkd-dy.jpg" alt="GKD-订阅">

## 3. 设置

这是个人在使用时候 **"设置"** 页面的一个截图。    

<img width=300px src="https://slab-1251708715.cos.ap-guangzhou.myqcloud.com/KGarden/2023/gkd-setting.jpg" alt="GKD-订阅">

**Comment by obaby** [comment]
首页的那个猫咪有意思～～

**Reply by shenweiyan** [reply]
👏👏
